org-ql.el

```diff
@@ -109,6 +109,10 @@ the value returned by it at that node.")
   "Plist of predicates, their corresponding functions, and their docstrings.
 This list should not contain any duplicates.")
 
+
+(defvar org-ql-default-predicate 'regexp
```

Comment:
I think we should name the variable `org-ql-plain-string-predicate`, which would be more descriptive.  The docstring should probably be something like, "Plain strings in queries are matched with this predicate."

Also, please avoid adding extra newlines (i.e. the one above `defvar`).

Reply:
changed

Reply:
is it a good idea to add document to helm-org-ql section?

Reply:
> is it a good idea to add document to helm-org-ql section?

Sorry, can you explain more what you mean?